src/remote-frame/index.js

```diff
@@ -79,6 +81,7 @@ const createCapturedContextComponent = (contextTypes = {}) => {
 class RemoteFrame extends Component {
   constructor(props, context) {
     super(props, context)
+    this.uniqueId = Date.now().toString(36) + Math.random().toString(36).substring(2);
```

Comment:
can we use uuid lib here?

Reply:
We can if we have a strong "opinion" why to add another library.
I think here we are not after the randomness provided by uuid library.
I think that max of 4-5 remote frames can be held in the stack and this is in my opinion enough code size to cover for that

Reply:
sure, was just a suggestion.